.changes/v1.40.0.md

```diff
@@ -0,0 +1,50 @@
+## v1.40.0 - 2023-02-01
+
+### 🎉 Feature
+
+- Add new flags to engine list command. Query flag for partial search, pool-name flag for search by pool name, sort flag for representing data in ascending or descending order, sorted-by flag for specifying field to sort.
+- Add `sort` flag for representing data in ascending or descending order. Add `sorted-by` flag for specifying which field to sort data on the pool list command.
+- Add `sort` flag for representing data in ascending or descending order. Add `sorted-by` flag for specifying which field to sort data on with the role list command.
+- Add `sort` flag to specify ascending or descending order to client search command.
+- Add `sort` flag for representing data in ascending or descending order. Add `sorted-by` flag for specifying which field to sort data on with the policy search command.
+- Add sort, sorted-by flags for representing data in ascending or descending order by some field with the user search command.
+- Add `sort` flag for representing data in ascending or descending order. Add `sorted-by` flag for specifying which field to sort data on with the group search command.
```

Comment:
Can we get this summarized as a single entry as it's very repetitive.

Reply:
I've tried